accumulo/templates/master_deployment.yaml

```diff
@@ -15,52 +15,52 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ include "accumulo.fullname" . }}-masters
+  name: {{ include "accumulo.fullname" . }}-managers
   labels:
     {{- include "accumulo.labels" . | nindent 4 }}
-    app.kubernetes.io/component: master
+    app.kubernetes.io/component: manager
 spec:
   replicas: 1
   selector:
     matchLabels:
       {{- include "accumulo.selectorLabels" . | nindent 6 }}
-      app.kubernetes.io/component: master
+      app.kubernetes.io/component: manager
   template:
     metadata:
       labels:
         {{- include "accumulo.labels" . | nindent 8 }}
-        app.kubernetes.io/component: master
+        app.kubernetes.io/component: manager
     spec:
       securityContext:
         runAsUser: 1001
       initContainers:
       - name: check-hadoop-jars
-        image: "{{ .Values.accumulo.initContainer.image }}:{{ .Values.accumulo.initContainer.tag }}"
+        image: "{{ .Values.initContainer.image }}:{{ .Values.initContainer.tag }}"
         securityContext:
           runAsUser: 1000
         command: 
         - "/bin/sh"
         args: ["-c", "while [ `hdfs dfs -ls hdfs://hdfs-nn:9000/datawave/accumulo-vfs-classpath | wc -l` -lt 5 ]; do echo \"Waiting for jars to load\"; sleep 10; done"]
```

Comment:
should this come from config as the namenode may not be here?

Reply:
I have debated this. One option is to leave this as is, and in the case the namenode is not in K8s, we would create an ExternalService so the naming is transparent to the apps in cluster